.github/workflows/automatic-merges.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: automatic-merges
+
+on:
+  workflow_run:
+    # If new workflows triggered on pull request are added, they should be added to this list
+    workflows:
+      - Link Checker
+      - yaml-lint
+      - dockerfile-lint
+      - groovy-tests
+      - manifests
+      - python-tests
+    types: completed
+
+jobs:
+  automatic-merge-manifests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: find-triggering-pr
+        uses: peternied/find-triggering-pr@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: peternied/discerning-merger@v1
+        if: steps.find-triggering-pr.outputs.pr-number != null
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          pull-request-number: ${{ steps.find-triggering-pr.outputs.pr-number }}
+          allowed-authors: |
+            dependabot
+            opensearch-trigger-bot
+            peterzhuamazon
+            bbarani
+            gaiksaya
+            rishabh6788
+            zelinh
+            jordarlu
+            prudhvigodithi
+            Divyaasm
+            dblock
+            tianleh
+            peternied
```

Comment:
How about dynamically get this list from https://github.com/opensearch-project/opensearch-build/blob/main/.github/CODEOWNERS similar to https://github.com/opensearch-project/opensearch-py/blob/main/.github/workflows/release-drafter.yml#L14-L16

Reply:
Hmm, that works well for repo maintainers, but I'd think this list might be larger since release owners are from all over the OpenSearch-Project.

Maybe this could be a follow up item?

Reply:
Sounds good!